src/components/title.scss

```diff
@@ -0,0 +1,14 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900);
```

Comment:
Du importerer Roboto-fonten i flere scss-filer. Kanskje det kunne vært lurt å definere denne i en egen fonts.scss, og så inkludere denne filen istedet? For jeg regner med at det er meningen at vi skal bruke samme font overalt på websiden, og ikke bare litt her og der?